features/README.md

```diff
@@ -6,7 +6,7 @@ specific topics.
 If you're reading this from the [generated HTML documentation on
 github.io](http://datastax.github.io/java-driver/), use the "Read More"
 menu on the right hand side. If you're [browsing the source files on
-github.com](https://github.com/datastax/java-driver/tree/2.2/features),
+github.com](https://github.com/datastax/java-driver/tree/3.0/features),
 simply navigate to each sub-directory.
 
 This is a work in progress: new sections will be added to cover existing
```

Comment:
The legacy user documentation link still references 2.2 as there is no 3.0 link (yet).